### PR TITLE
pscanAlpha Title Caps Fixes

### PR DESCRIPTION
**Pull request by kingthorin**
Per:
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#style-guidelines

Related to zaproxy/zaproxy#2000

### CODE REVIEW

**Comment by psiinon** [comment]
Looks good.

File: src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html [hunk 1]
```diff
@@ -46,15 +46,15 @@
...
 This check looks for insecure HTTP pages that host HTTPS forms. The issue is that an insecure HTTP page can easily be hijacked through MITM and the secure HTTPS form can be replaced or spoofed.
 
-<H2>HTTPS to HTTP insecure transition in form post</H2>
+<H2>HTTPS to HTTP Insecure Transition in Form Post</H2>
 This check identifies secure HTTPS pages that host insecure HTTP forms. The issue is that a secure page is transitioning to an insecure page when data is uploaded through a form. The user may think they're submitting data to a secure page when in fact they are not.
 
 <H2>Insecure Component</H2>
-Passively scans the server response headers and body generator content for product versions, which are then cross-referenced against a list of product versions known to be vulnerable to various CVEs.<br>
-A list of ranked CVEs and CVSS severity scores is output for each product noted to be vulnerable.<br>
+Passively scans the server response headers and body generator content for product versions, which are then cross-referenced against a list of product versions known to be vulnerable to various CVEs.
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html]: This changes were on purpose, right?

Comment (kingthorin) [inline src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html]: Ya the forced line feeds didn't make sense when reviewing the rendered version of the page.

Comment (thc202) [inline src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html]: OK, just wanted to confirm that.

**Comment by thc202** [comment]
Thanks.

**Pull request merged**